redash/tasks/queries/execution.py

```diff
@@ -204,12 +204,10 @@ def run(self):
         if error is not None and data is None:
             result = QueryExecutionError(error)
             if self.is_scheduled_query:
-                self.query_model = models.db.session.merge(self.query_model, load=False)
```

Comment:
I missed these updates... Could you let me know why you removed these lines (L207 and L212)?

Reply:
restored it

Reply:
The one of L212 was restored. However, it looks like that L207 (after `if self.is_scheduled_query:`) is still removed. Please check.

Reply:
diff is not displayed properly, you can check it a branch of my fork

Reply:
found it and fixed

Reply:
Thanks. Confirmed.